com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -7,18 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
-## [Unreleased]
 
-### Fixed
-
-- Implicit conversion of NetworkObjectReference to GameObject will now return null instead of throwing an exception if the referenced object could not be found (i.e., was already despawned) (#2158)
-- Fixed warning resulting from a stray NetworkAnimator.meta file (#2153)
-- Fixed ClientRpcs always reporting in the profiler view as going to all clients, even when limited to a subset of clients by `ClientRpcParams`. (#2144)
-- Fixed RPC codegen failing to choose the correct extension methods for `FastBufferReader` and `FastBufferWriter` when the parameters were a generic type (i.e., List<int>) and extensions for multiple instantiations of that type have been defined (i.e., List<int> and List<string>) (#2142)
-- Fixed throwing an exception in `OnNetworkUpdate` causing other `OnNetworkUpdate` calls to not be executed. (#1739)
-
-## [1.0.2] - 2022-09-XX
+## [1.0.2] - 2022-09-12
 
+- Changed version to 1.0.2. (#2170)
```

Comment:
do we even need this entry here? (isn't the title tag obvious?)

Reply:
removed